main.cpp

```diff
@@ -84,42 +104,92 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
 /*
  1)
  */
+float throwHammer(double hammerWeight, double power)
+{
+    ignoreUnused(hammerWeight, power);
+    return {};
+}
 
 /*
  2)
  */
+bool isStuck(bool canMove = false, bool canSee = false, bool canHear = false)
+{
+    ignoreUnused(canMove, canSee, canHear);
+    return {};
+}
 
 /*
  3)
  */
+int yourChange(double cost, double paid, int dollars = 0, int quarters = 0, int dimes = 0)
+{
+    ignoreUnused(cost, paid, dollars, quarters, dimes);
+    return {};
+}
 
 /*
  4)
  */
+float explode(double temp, double pressure, bool fracture = false)
+{
+    ignoreUnused(temp, pressure, fracture);
+    return {};
+}
 
 /*
  5)
  */
+char letterGrade(int totalPoints, int possiblePoints, bool wasLate = false)
+{
+    ignoreUnused(totalPoints, possiblePoints, wasLate);
+    return {};
+}
 
 /*
  6)
  */
+unsigned int elevatorLight(int floor, bool doorsClosed, char inspectorGrade)
+{
+    ignoreUnused(floor, doorsClosed, inspectorGrade);
+    return {};
+}
 
 /*
  7)
  */
+unsigned int liftWeights(int weight, int reps, bool wearHeadphones = false)
+{
+    ignoreUnused(weight, reps, wearHeadphones);
+    return {};
+}
 
 /*
  8)
  */
+double getAudited(bool isSame, double income, double deductions, float taxRate = 10.0f)
+{
+    ignoreUnused(isSame, income, deductions, taxRate);
+    return {};
+}
 
 /*
  9)
  */
+int spinWheel(bool coinInserted = false, char buttonSelect = 'a', double pullLever = 2.2)
+{
+    ignoreUnused(coinInserted, buttonSelect, pullLever);
+    return {};
+}
 
 /*
  10)
  */
+float laughter(unsigned int funny, unsigned int numDrinks)
```

Comment:
This function name does not contain a verb.
Function names need to describe actions
- [ ] revise